src/calculateRentalCost.js

```diff
@@ -4,7 +4,27 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40; // dolars$
+  const longTerm = 7; // days
+  const longTermDiscount = 50; // dolars$
+  const shortTerm = 3; // days
+  const shortTermDiscount = 20; // dolars$
+  let total = days * pricePerDay;
```

Comment:
why let? there should be const
```suggestion
  const total = days * pricePerDay;
```
total what? naming should be better like totalRentCost or just result ;)

Reply:
Fixed - changes implemented in commit 3e47e5e